pkg/nodejs/npm/parse_testcase.go

```diff
@@ -8,9 +8,9 @@ var (
 	// npm install --save promise jquery
 	// npm ls | grep -E -o "\S+@\S+" | awk -F@ 'NR>0 {printf("{\""$1"\", \""$2"\", \"\"},\n")}'
```

Comment:
We need to fix the command as well. Otherwise, you have to edit these values manually.